docs/OV_Runtime_UG/auto_device_selection.md

```diff
@@ -290,93 +288,92 @@ The property enables you to control the priorities of networks in the Auto-Devic
+      compiled_model1 = core.compile_model(model=model, device_name="AUTO:GPU,MYRIAD,CPU", config={"MODEL_PRIORITY":"MEDIUM"})
+      compiled_model2 = core.compile_model(model=model, device_name="AUTO:GPU,MYRIAD,CPU", config={"MODEL_PRIORITY":"LOW"})
 
       # Assume that all the devices (CPU, GPU, and MYRIAD) can support all the networks.
       # Result: compiled_model0 will use GPU, compiled_model1 will use MYRIAD, compiled_model3 will use CPU.
-      
+
       # Example 2
       # Compile and load networks:
-      compiled_model0 = core.compile_model(model=model, device_name="AUTO:CPU,GPU,MYRIAD", config={"AUTO_NETWORK_PRIORITY":"2"})
-      compiled_model1 = core.compile_model(model=model, device_name="AUTO:CPU,GPU,MYRIAD", config={"AUTO_NETWORK_PRIORITY":"1"})
-      compiled_model2 = core.compile_model(model=model, device_name="AUTO:CPU,GPU,MYRIAD", config={"AUTO_NETWORK_PRIORITY":"2"})
+      compiled_model0 = core.compile_model(model=model, device_name="AUTO:GPU,MYRIAD,CPU", config={"MODEL_PRIORITY":"LOW"})
+      compiled_model1 = core.compile_model(model=model, device_name="AUTO:GPU,MYRIAD,CPU", config={"MODEL_PRIORITY":"MEDIUM"})
+      compiled_model2 = core.compile_model(model=model, device_name="AUTO:GPU,MYRIAD,CPU", config={"MODEL_PRIORITY":"LOW"})
 
       # Assume that all the devices (CPU, GPU, and MYRIAD) can support all the networks.
       # Result: compiled_model0 will use GPU, compiled_model1 will use GPU, compiled_model3 will use MYRIAD.
 @endsphinxdirective
 
 ## Configuring Individual Devices and Creating the Auto-Device plugin on Top
-Although the methods described above are currently the preferred way to execute inference with AUTO, the following steps can be also used as an alternative. It is currently available as a legacy feature and used if the device candidate list includes VPUX or Myriad (devices uncapable of utilizing the Performance Hints option). 
+Although the methods described above are currently the preferred way to execute inference with AUTO, the following steps can be also used as an alternative. It is currently available as a legacy feature and used if the device candidate list includes Myriad (devices uncapable of utilizing the Performance Hints option). 
 
 @sphinxdirective
 .. tab:: C++ API
 
    .. code-block:: cpp
 
-      ovCore core;
+      ov::Core core;
 
       // Read a network in IR, PaddlePaddle, or ONNX format
-      stdshared_ptrovModel model = core.read_model(sample.xml);
+      std::shared_ptr<ov::Model> model = core.read_model("sample.xml");
 
-      // Configure the VPUX and the Myriad devices separately and load the network to the Auto-Device plugin
-      set VPU config
-      core.set_property(VPUX, {});
+      // Configure the CPU and the Myriad devices separately and load the network to the Auto-Device plugin
+      // set CPU config
+      core.set_property("CPU", {});
```

Comment:
we can also use `ov::device_properties` (see openvino_docs_OV_UG_Hetero_execution.html#configure-fallback-devices)

Reply:
fixed it